test/bootswatch4_test.js

```diff
@@ -39,6 +39,16 @@ describe('bootswatch4', () => {
         helpers.assert.bodyClass('page-bootswatch', response, done);
     });
 
+    it('handles invalid theme query parameter', (done) => {
```

Comment:
You could toss this inside of an array that tests for all the possible invalid options. 
```
[ -1, 500, 'foobar' ].forEach((i) => {
    it('handles invalid theme query parameter (' + i + ')', (done) => {
        // test here
    });
});